packages/sources/web-components/src/components/vtmn-link/vtmn-link.tsx

```diff
@@ -0,0 +1,49 @@
+import { Component, Prop, h } from '@stencil/core';
+
+@Component({
+  tag: 'vtmn-link',
+  styleUrl: 'vtmn-link.pcss',
+  shadow: true,
+})
+export class VtmnLink {
+  /**
+   * The size of the link.
+   * @default 'medium'
+   */
+  @Prop() size: 'small' | 'medium' | 'large' = 'medium';
+
+  /**
+   * Is the link standalone or not
+   * @default false
+   */
+  @Prop({ attribute: 'standalone' }) isStandalone: boolean = false;
```

Comment:
```suggestion
  @Prop() isStandalone: boolean = false;
```
Like we did with React, it's okay.